### PR TITLE
fix: project description - handle whitespace (APPLICS-311)

### PR DESCRIPTION
**Pull request by BenSurgisonPINS**
https://pins-ds.atlassian.net/browse/APPLICS-311

- convert line-breaks to <br> tags to preserve formatting in the summary field within the application data
- add the safe filter to the about.njk summary field to allow the <br> tags 
- add a unit test to prove the functionality works

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/_utils/get-application-data.js [hunk 1]
```diff
@@ -17,12 +19,16 @@ const getApplicationData = async (case_ref) => {
 
 	const DateOfDCOSubmission = badDateToNull(data.DateOfDCOSubmission);
 
+	if (data.Summary) {
+		console.log('XXXXXXXXXXXXXXX');
```
Comment (Yakitrak) [inline packages/forms-web-app/src/pages/projects/_utils/get-application-data.js:23]: Forgot to remove this? :D

File: packages/forms-web-app/src/pages/projects/_utils/get-application-data.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ const badDateToNull = (date) => (date === '0000-00-00' ? null : date);
 
 const add28DaysToDate = (date) => (date ? dayjs(date).add(28, 'days').toISOString() : null);
 
+const preserveLinebreaks = (text) => text.replace(/\r\n|\r|\n/g, '<br>');
```
Comment (Yakitrak) [inline packages/forms-web-app/src/pages/projects/_utils/get-application-data.js:9]: As we are using this in multiple places, can we move this to a common export and import everywhere?

Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/pages/projects/_utils/get-application-data.js:9]: It's only used here at the moment, but I can see it being useful elsewhere.  Perhaps the other functions above (badDateToNull and add28DaysToDate) could do with being moved into a utils as well.  This was the reason I left this one in situ as well as these were already here.

Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/pages/projects/_utils/get-application-data.js:9]: Should this be another ticket to consolidate these functions?

File: packages/forms-web-app/src/pages/projects/_utils/get-application-data.js (hunk 1 above)
Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/pages/projects/_utils/get-application-data.js]: I've removed it, thanks

File: packages/forms-web-app/src/pages/projects/_utils/get-application-data.js (hunk 2 above)
Comment (BenSurgisonPINS) [reply packages/forms-web-app/src/pages/projects/_utils/get-application-data.js]: I have created a new reusable function as requested.

**Review by cprieststephens — COMMENTED** [review]
Looks good to me.

**Review by Yakitrak — APPROVED** [review]

**Pull request merged**